modules/test/playwright/tests/locked-items-web/pages/LockedItems.page.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {ProductMenuPage} from '../../../pages/product-navigation-product-menu/ProductMenu.page';
+
+export class LockedItemsPage {
+	readonly page: Page;
+	readonly lockedPagesMenuItem: Locator;
+	readonly lockedPagesTitle: Locator;
+	readonly pageTitle: Locator;
+	readonly productMenuPage: ProductMenuPage;
+
+	constructor(page: Page) {
+		this.page = page;
+		this.lockedPagesMenuItem = page.getByRole('menuitem', {
+			name: 'Pages',
+		});
+		this.lockedPagesTitle = page
+			.locator('p.sheet-title')
```

Comment:
Looks like a weak locator 😓

Reply:
I haven't found another stronger locator. Maybe it is a good idea to add a test ID ( see [docs](https://playwright.dev/docs/api/class-page#page-get-by-test-id) )

Reply:
nope, add a testID IMHO should be avoided as much as possible

Reply:
So, any proposal? Or do I leave it like this?

Reply:
Finally the locator should be `p.sheet-title` because `sheet-title` is failing